EIP-7777-SuperFuse.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7777
```

Comment:
```suggestion
eip: 8044
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.  

Please also update the filename.